python/taichi/ui/scene.py

```diff
@@ -205,6 +206,9 @@ def mesh(self,
             index_count (int, optional):
                 only available when `indices` is provided, which is the the number
                 of vertices to draw.
+            display_mode (int, optional):
```

Comment:
It's really error prone and hard for users to memorize these, try using an enum instead of ints?

Reply:
Yeah, I add a new class named DisplayMode to support passing enum numbers in newer updates.